src/seedu/addressbook/data/person/Name.java

```diff
@@ -40,7 +40,7 @@ public static boolean isValidName(String test) {
      * Retrieves a listing of every word in the name, in order.
      */
     public List<String> getWordsInName() {
-        return Arrays.asList(fullName.split("\\s+"));
+        return Arrays.asList(fullName.toLowerCase().split("\\s+"));
```

Comment:
You might want to look into whether [Java String APIs](https://docs.oracle.com/javase/9/docs/api/java/lang/String.html#equalsIgnoreCase-java.lang.String-) can help you to do this